locations/spiders/barlouie.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+import scrapy
+
+from locations.items import GeojsonPointItem
+
+
+class BarlouieSpider(scrapy.Spider):
+    name = "barlouie"
+    allowed_domains = ["www.barlouie.com"]
+    start_urls = ["https://www.barlouie.com/locations"]
+    item_attributes = {"brand": "Barlouie", "brand_wikidata": "Q16935493"}
+
+    def parse(self, response):
+        for link in response.xpath("//a[contains(@href, 'locations')]/@href").getall():
```

Comment:
Looked for adding a rule like
```
    rules = [
        Rule(
            LinkExtractor(allow=r"store\?StoreID=(\d+)$"), callback="parse_each_website"
        )
    ]
```
so that this function could have been removed. but couldn't find the right regex.
Please let me know if this can be done.

Reply:
Looks like it can be a SitemapSpider with
```
sitemap_urls=["https://www.barlouie.com/sitemap.xml"]
sitemap_rules=[("/locations/", "parse_each_website")]
```

Reply:
thanks.